.github/workflows/main.yml

```diff
@@ -99,11 +102,24 @@ jobs:
       SYNAPSE_PRINCIPAL_PASSWORD: ${{ secrets.IE_SYNAPSE_PRINCIPAL_PASSWORD }}
     needs: build
 
+  build_php_storage_driver_common:
+    uses: ./.github/workflows/build-php-storage-driver-common.yml
+    with:
+      hasCodeChanged: ${{ needs.build.outputs.changed-php-storage-driver-common == 'true' }}
+      isTag: ${{ startsWith(github.ref, 'refs/tags/') }}
+      BUILD_AWS_ACCESS_KEY_ID: AKIAQ4QRYXTAJQ32HBOK # this sucks, but can't use env here. See https://github.com/orgs/community/discussions/26671#discussioncomment-3252793
```

Comment:
:face_with_spiral_eyes:

Reply:
To je takto vsade ale v dalsom PR to mazem a nahradzujem tou rolou necchel som to tu komplikovat :D